resolvconf.in

```diff
@@ -523,7 +525,8 @@ list_resolv()
 	fi
 
 	cd "$IFACEDIR"
-	retval=1
+	retval=0
+	[ -n "$pattern_specified" ] && retval=1
```

Comment:
I think this needs to be more clear as to the intent.

Can you change this to somthing like this please?
```
# Return an error only if we have a pattern to match with no results
if [ -n "pattern_specified" ]; then
     retval=1
else
     retval=0
fi
```

This also reduces the number of assignments on the more common path of supplying a pattern.